lensless/benchmark.py

```diff
@@ -147,6 +148,75 @@ def __getitem__(self, idx):
         return lensless, lensed
 
 
+class BenchmarkDataset(ParallelDataset):
```

Comment:
Do we need this object? 

I would either:
- rename to `DiffuserCamDataset` so it's specific to that, or
- remove this and default `ParallelDataset`  for the DiffuserCam dataset (as it's already done, but add the defaults for the transforms the PSF, downsampling, etc)

Reply:
and downloading (if necessary)

Reply:
renaming `DiffuserCamTestDataset`